src/watchmaker/workers/salt.py

```diff
@@ -266,6 +276,24 @@ def _build_salt_formula(self, extract_dir):
                 filepath=salt_content_file,
                 to_directory=extract_dir
             )
+            if self.salt_content_path:
+                self.log.debug(
+                    'Using salt content path: %s',
+                    self.salt_content_path
+                )
+                try:
+                    salt_content_src = os.sep.join((
+                        extract_dir, self.salt_content_path))
+                    salt_file_names = os.listdir(salt_content_src)
+                    for name in salt_file_names:
```

Comment:
after we're done here, we can also remove the `salt_content_src` directory...

```suggestion
                    for name in salt_file_names:
                        srcname = os.sep.join((salt_content_src, name))
                        shutil.move(srcname, extract_dir)

                    shutil.rmtree(salt_content_src)
```

Reply:
roger! will do that.